src/Formality__Form.re

```diff
@@ -143,65 +156,51 @@ module Make = (Form: Form) => {
         switch (validator) {
         | None => React.Update({...state, data})
         | Some(validator) =>
-          let emitted = state.emittedFields->Set.has(field);
-          switch (validator.strategy, emitted, state.submittedOnce) {
```

Comment:
By removing `emitted` we're loosing information if validation result for current field was ever shown in UI. It breaks some strategies. Just FYI, since it's going to be removed anyway w/ updated `status` type.